app/build.gradle

```diff
@@ -13,7 +13,6 @@ android {
         targetSdk 33
         versionCode 9
         versionName "1.2.4"
-        resourceConfigurations += ["en", "zh_CN"]
```

Comment:
Can you describe a little bit more about why these configurations are not needed anymore?

Reply:
https://developer.android.com/guide/topics/resources/app-languages#gradle-config

this line add by my mistake and it will cause problem if new language been add